src/NServiceBus.Persistence.AzureTable/Obsoletes.cs

```diff
@@ -50,5 +50,12 @@ public static partial class ConfigureAzureSubscriptionStorage
             object client) =>
             throw new NotImplementedException();
     }
+
+    public partial class CompatibilitySettings
+    {
+        [ObsoleteEx(Message = "The compatibility mode has been disabled and requires opt-in to enable. Calling this method is no longer necessary.", TreatAsErrorFromVersion = "5.0.0", RemoveInVersion = "6.0.0")]
```

Comment:
```suggestion
        [ObsoleteEx(Message = "The compatibility mode has been disabled by default and requires opt-in to enable. Calling this method is no longer necessary.", TreatAsErrorFromVersion = "5.0.0", RemoveInVersion = "6.0.0")]
```

Reply:
Merging the suggestion and will then amend to the commit the approval tests